scripts/buildTypes.mts

```diff
@@ -62,25 +66,30 @@ async function copyDeclarations(sourceDirectory: string, destinationDirectory: s
 
 interface HandlerArgv {
   skipTsc: boolean;
+  copy: string[];
 }
 
 async function main(argv: HandlerArgv) {
   const packageRoot = process.cwd();
+  const tsconfigPath = path.join(packageRoot, 'tsconfig.build.json');
```

Comment:
Alternatively https://www.npmjs.com/package/tsconfig

Reply:
Or [tsconfck](https://github.com/dominikg/tsconfck). But seems overkill, unless we want a resolved tsconfig which also goes through the `extends` logic.

Reply:
> unless we want a resolved tsconfig

We're reading properties out of it, so I guess we kind of do.
But it's also something we can add in follow-up.